### DISCUSSION TITLE
controlling the size of ViewFinder

### DISCUSSION THREAD

**Discussion by schlauerwolf** [mebjas/html5-qrcode#477] [Q&A]
Hi
how to I control the size (width & heigth) video stream using Html5Qrcode.
I tried html5QrCode.applyVideoConstraints to no avail.

Thank you !

**Comment by TGudat** [comment]
Hi
Is there any update on this? I am having the same problem, that the resulting video stream is only 300px wide and I didn't find a way to change that.

Thanks

**Comment by ROBERT-MCDOWELL** [comment]
you should learn the code and fix it by yourself. btw if no one answered your issue is elsewhere out of the project. Many causes can do this issue and without more info you will never have a solution unless you try to understand the code  behind the process.
And again, say "any update on this" will stress more than give you the answer since a git repo IS NOT a technical support after sale.

**Reply by TGudat** [reply]
Hi Robert,
I've spent a quit some time trying to understand the documentation of this library and changing my code. I've also tried altering code outside of this project, with no success. The most likely reason is my lack of expertise, which includes not knowing if I'm missing something within this library, something unrelated to this project or there actually being an issue. 
Since somebody asked the same question I have in the Q&A section it seems like a good place to ask. Maybe the creator found an answer I can benefit from or this question is something to look into. 
Not everybody can learn the code and fix all of their problems by themselves.

**Comment by ROBERT-MCDOWELL** [comment]
> Not everybody can learn the code and fix all of their problems by themselves

well, if we want this world to change for the best you should to..... and it's the same for all inidivdual needs.
